vcluster/manage/sleep-wakeup.mdx

```diff
@@ -1,12 +1,15 @@
 ---
-title: Sleep & Wakeup vCluster
-sidebar_label: Sleep & Wakeup
+title: Sleep and wakeup vCluster
+sidebar_label: Sleep and Wakeup
 sidebar_position: 4
 ---
 import BasePrerequisites from '@site/docs/_partials/base-prerequisites.mdx';
 import InstallCli from '../_partials/deploy/install-cli.mdx';
 
-The sleep mode **temporarily scales down** a virtual cluster and deletes all its created workloads on the host cluster. This approach helps **save computing resources** used by virtual cluster workloads in the host cluster. For example, sleeping development clusters during non-working hours can significantly reduce infrastructure costs.
+The sleep mode **temporarily scales down** a virtual cluster and deletes all its created workloads on the host cluster. This approach helps **save computing resources** used by virtual cluster workloads in the host cluster. For example, sleeping development clusters during non-working hours can significantly reduce infrastructure costs. 
+
+For more details on configuring sleep mode operations in vCluster, see the [Sleep mode documentation](../../configure/vcluster-yaml/sleep-mode).
```

Comment:
```suggestion
For more details on automating sleep mode operations in vCluster, see the [Sleep mode documentation](../../configure/vcluster-yaml/sleep-mode).
```
"Sleep mode" is our automatic solution, but there is also manual sleep/wake, so good to specify difference.